sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,16 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self.args[0].is_real:
+            return True
```

Comment:
~~Consider that `cosh(z).is_positive` where `z` is a complex number, that would be equivalent to solving the complex inequality `solveset(cosh(z) > 0, z)` (try this in SymPy) gives "inequalities not supported in complex domain".~~
I think @jksuom is right :)